packages/styled-ui-docs/components/CodeBlock.jsx

```diff
@@ -10,7 +10,7 @@ import * as ReactTable from 'react-table';
 import { AutoSizer } from 'react-virtualized';
 import { codeBlockLight, codeBlockDark } from '../prism-themes/styled-ui';
 import FontAwesomeIcon from './FontAwesomeIcon';
-import TMIcon from './TMIcon';
+import iconList from './tmicon-list';
```

Comment:
I am wondering if we can reuse `tmiconSvgMap` so that we don't have to maintain two similar copies of icon source. The downside of reusing `tmiconSvgMap` is that we will bundle it into `react-styled-ui` in order to use it in doc.

Reply:
你可以把 tmIconMap 和 tmIconSvgMap 分成兩個不同的檔案

* tmIconMap.js -> icon list
* tmIconSvgMap.js -> used to compose React components

然後在 docs 用 relative import

```js
import tmIconMap from '../../react-styled-ui/src/TMIcon/tmIconMap';
```

Reply:
@austinhung81 @ultralabed @joshuatai 

以後也許可以這樣，把那些 icon list 放在新的 package

Example:

```
packages/tonic-ui-icons
```

這樣就可以 share 同一份 source 了，然後 `tonic-ui-icons` 也有自己的 build script

Reply:
@joshuatai @austinhung81 @tinaClin @roth1002 @freewill0918 

1.0 之後如果這些都叫做 Tonic Icon，那應該是直接用 `<Icon />` 就好，不需要多一個 `<TMIcon />`

Reply:
就叫 <Icon /> 吧

Reply:
Well noted. Changes below will be made.
1. Create `tmIconMap.js` to host icon configurations and details.
2. Create `tmIconSvgMap.js` to compose React SVG components.
3. Rename `TMIcon` to `Icon`.